resources/components/views/Charts.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+    <LayoutBasic>
+        <div class="mb-4">
+            <div id="daterange" class="dropdown float-end">
+                <button
+                    id="dateRangeDropdown"
+                    aria-expanded="false"
+                    class="btn btn-primary dropdown-toggle"
+                    data-mdb-toggle="dropdown"
+                    type="button"
+                >
+                    <i aria-hidden="true" class="fas fa-calendar"></i>&nbsp;
+                    {{ i18n.get('_.stats.range') }} ({{ this.dateRange }})
+                </button>
+                <ul aria-labelledby="dateRangeDropdown" class="dropdown-menu">
+                    <li><a class="dropdown-item" href="#"
+                           @click.prevent="fetchRecentDays(7)">{{
+                            i18n.choice('_.stats.range.days', 1, {"days": 7})
+                        }}</a>
+                    </li>
+                    <li><a class="dropdown-item" href="#"
+                           @click.prevent="fetchRecentDays(15)">{{
+                            i18n.choice('_.stats.range.days', 1, {"days": 15})
+                        }}</a>
```

Comment:
why 15 days? Isn't 7, 14, 30, 60, 90, ... the normal range?

Reply:
True